brayns/Brayns.cpp

```diff
@@ -835,6 +832,9 @@ struct Brayns::Impl : public PluginAPI
         app.setSynchronousMode(!app.getSynchronousMode());
     }
 
+    std::vector<DynamicLib> _pluginLibs;
+    std::vector<ExtensionPluginPtr> _extensions;
```

Comment:
I would move the 2 loadPlugin methods and these members inside the ExtensionPluginFactory (could be renamed to something nicer) instead of removing the class

Reply:
Hmm, maybe at a later stage because this is evolving. I need to see how it works with command line options before reviving it.